Sources/MapboxNavigation/CarPlayManagerDelegate.swift

```diff
@@ -884,6 +894,14 @@ public extension CarPlayManagerDelegate {
                         in mapTemplate: CPMapTemplate) -> Bool {
         return false
     }
+
+    /**
+     `UnimplementedLogging` prints a warning to standard output the first time this method is called.
+     */
+    func carPlayNavigationViewController(_ carPlayManager: CarPlayManager,
+                                         guidanceBackgroundColor style: UIUserInterfaceStyle) -> UIColor? {
+        return nil
```

Comment:
```suggestion
        logUnimplemented(protocolType: CarPlayManagerDelegate.self, level: .debug)
        return nil
```
I think we also forgot to add it in the three methods before, could you update when you have a chance?